Cargo.toml

```diff
@@ -11,7 +11,7 @@ rust-version = "1.60.0"
 
 [dependencies]
 arrayvec = { version = "0.7", default-features = false }
-serde = { version = "1.0.183", optional = true }
+serde = { version = "1.0.183", default-features = false, features = ["alloc", "derive"], optional = true }
```

Comment:
So we dont actually need the `alloc` and `derive` features at all?

Reply:
Interesting, potentially no. This got me looking into how `serde` is used and it looks like it's actually gated by the `std` feature?
https://github.com/paritytech/parity-scale-codec/blob/master/src/compact.rs#L195-L196
Seems like that should change

Reply:
I pushed a change that always includes `serde`, even if `std` isn't enabled. I think that should work...